test/Shippo/AddressTest.php

```diff
@@ -7,6 +7,27 @@ public function testValidCreate()
         $address = self::getDefaultAddress();
         $this->assertEqual($address->object_state, 'VALID');
     }
+
+    public function testResidentialCreate()
+    {
+        $address = Shippo_Address::create(array(
+            'object_purpose' => 'QUOTE',
```

Comment:
can you change `QUOTE` to `PURCHASE`. The first one will be deprecated eventually.